src/colorizer/types.ts

```diff
@@ -319,12 +319,16 @@ export enum LoadTroubleshooting {
     " Please check your network access.",
   CHECK_FILE_EXISTS = "Please check if the file exists and if you have access to it, or see the developer console for more details.",
   CHECK_FILE_OR_NETWORK = "This may be because of an unsupported format, missing files, or server and network issues. Please see the developer console for more details.",
+  CHECK_ZIP_FORMAT = "Make sure a 'collection.json' or 'manifest.json' file exists in the root directory.",
+  CHECK_ZIP_FORMAT_COLLECTION = "A 'collection.json' should exist in the root directory.",
+  CHECK_ZIP_FORMAT_MANIFEST = "A 'manifest.json' should exist in the root directory.",
 }
 
 export enum LoadErrorMessage {
```

Comment:
Could be a `const enum`? Benefits and drawbacks:

- benefit: enum is inlined rather than becoming a weird object in the final code
- drawback: these strings are all pretty large, if one or two are used twice the cost of inlining them probably overrides the weird object thing.

Probably best to keep these non-`const` for now, but I have a potentially unjustified vendetta against non-`const` enums. See next comment for how they might make sense anyways?